src/modeler/AutoRest.Swagger/Validation/Core/RulesExtensions.cs

```diff
@@ -64,6 +64,6 @@ public static IEnumerable<Rule> GetValidationRules(this Type type)
         /// <summary>
         /// The collection of rules that apply to all properties regardless of other rules.
         /// </summary>
-        public static IEnumerable<Rule> UniversalRules = new[] { new NoControlCharacters() };
+        public static IEnumerable<Rule> UniversalRules = new List<Rule>();
```

Comment:
so we don't have universal rules afterwards? In that case I'd remove this entirely (and maybe some places where it's used get cleaner) since we shouldn't ever add a universal rule.
Not only because it's way simpler in TS, I'm also just super skeptical about "universal". In case of this rule, even control characters may have their place in Swaggers, e.g. in `examples` sections where you can have raw response bodies where control characters are perfectly valid.